spanner/api/Spanner/Program.cs

```diff
@@ -1343,6 +1378,177 @@ await connection.CreateDdlCommand($"DROP TABLE {table}")
             return ExitCode.Success;
         }
 
+        public static object BatchInsertRecords(string projectId,
+            string instanceId, string databaseId)
+        {
+            var response =
```

Comment:
I think developers can find this confusing.
By Microsoft conventions, when the method is called XXXAsync, it returns a Task or Task<T>. Therefore, if the method is async, you can do <b>var response = await BatchInsertRecordAsync(...);</b> and the expectation is that the method will return the T that Task running the async method returns.
In your case you write the method as synchronous, therefore, it should be <b>var response<i>Task</i> = BatchInsertRecordsAsync...</b>

Reply:
Done.